config/software/chef-gem.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 name "chef-gem"
-default_version "11.12.2"
+default_version "11.18.2"
```

Comment:
11.18.12 plus "lolwut" i assume we think this is still used by chef 11.x builds?